### DISCUSSION TITLE
list of known working SDR as eNB in LTE

### DISCUSSION THREAD

**Discussion by salacate** [srsran/srsRAN_Project#257] [General Help] [answered]
Hello everyone ,i was planning on doing a private LTE network using SDR and srsRAN ,but after research it got kinda confusing on what SDR can work as eNB ,some saying that  pluto SDR and even hackRF can work as a eNB and some saying only limeSDR ,BladeRF ,and usrp can work because of the Hardware timestamping feature 
so my question is does anyone know what are the SDR that can work as eNB for a private LTE network ,im looking for a budget SDR and the latter SDR mentioned are kinda expensive ,apricate any help

**Comment by brendan-mcauliffe** [comment, marked as answer]
Hi @salacate, 

First off, as this is a question relating to LTE, and thus the srsRAN 4G code base, it would be better to ask this question on the srsRAN 4G discussion board. 

I'll try to answer your question here anyway. I suggest you take a look at [this page](https://docs.srsran.com/projects/4g/en/latest/app_notes/source/hw_packs/source/index.html) in the srsRAN 4G documentation. It outlines some potential hardware set-ups (PC and SDR) for use with srsRAN 4G. In general, the SDR you choose will ultimately depend on your use case. You should check that the SDR you choose is capable of operating in the frequency range of your use case, can sustain the bandwidth you require, and has the correct number of channels if you intend to do something like MIMO or carrier aggregation. As with anything, the lower in price you go the fewer features you will have, so a minimum set of requirements is something you will need to identify. You should also make sure your PC is powerful enough, as no matter how good your USRP is, if your PC is not powerful enough, it will cause a bottleneck and severely limit your performance. 

All of this is outlined in the document I shared. If you have any further questions it would be great if you could start a thread [here](https://github.com/srsran/srsRAN_4G/discussions).

**Reply by salacate** [reply]
hello again ,thanks for the response ,will try to ask on that thread didn't know there was a thread for  4G ,apricate the help
